internal/cli/users.go

```diff
@@ -322,7 +324,9 @@ func deleteUserCmd(cli *cli) *cobra.Command {
 		Use:   "delete",
 		Args:  cobra.MaximumNArgs(1),
 		Short: "Delete a user",
-		Long:  "Delete a user.",
+		Long: "Delete a user.\n\n" +
+			"To delete interactively, use `auth0 users delete` with no arguments.\n\n" +
+			"To delete non-interactively, supply the user id and the `--force` flag to skip confirmation.",
 		Example: `auth0 users delete 
 auth0 users delete <id>`,
```

Comment:
Consider adding the `auth0 users delete <id> --force` example in a future PR, like it is in other commands.